tests/network/test_network_actors.py

```diff
@@ -91,8 +96,10 @@ def test_alice_finds_ursula(alice, ursulas):
     getter = alice.server.get(all_ursulas[ursula_index])
     loop = asyncio.get_event_loop()
     value = loop.run_until_complete(getter)
-    _signature, _ursula_pubkey_sig, _hrac, interface_info = dht_value_splitter(value.lstrip(b"uaddr-"),
```

Comment:
+1; Awesome work finding this bug.

Reply:
✔️

Reply:
Also, +1. This was a tricky find. Good job!

Reply:
✔️